index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+        </div>
+     </section>
+
+     <section id="about-me">
+        <h1>About me</h1>
+        <div class="img3"></div>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me. 
+        </p>
+        <div class="img4"></div>
+        <button class="btngreen">
+            Get My Resume
+        </button>
+     </section>
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038811045) from the previous reviewer. 

![Screenshot from 2022-12-04 01-20-33](https://user-images.githubusercontent.com/60083437/205458129-c536b04f-7dba-430e-9833-03fa98be352d.png)